source/documentation/rshiny-app.md

```diff
@@ -78,21 +78,17 @@ Your app can take one of several forms:
 
 By default, the template contains `server.R` and `ui.R` files. However, you may wish to take a different approach depending on your requirements. For example, using `app.R`, it is possible to deploy RShiny apps from within a package, as in this [example from the costmodelr package](https://github.com/RobinL/costmodelr/blob/b328902026bd1cce5d17b487e310c59725ea4d62/R/shiny_explorer.r#L20).
 
-### Manage dependencies
+### List your package dependencies
 
-Most apps will have dependencies on various third-party packages (e.g., `dplyr`). These packages change through time and may not always be backwards compatible. To avoid compatibility issues and ensure reproducible outputs, it is necessary to use a package management system, such as packrat, renv, or conda.
+Most apps will have dependencies on various third-party packages (e.g., `dplyr`).
 
-You can find further guidance in the [package management](https://user-guidance.services.alpha.mojanalytics.xyz/appendix/conda/#package-management) section.
+In your project repo you will need a file that lists your dependencies. We use 'package management' to management these lists. The packages change through time and may not always be backwards compatible. To avoid compatibility issues and ensure reproducible outputs, it is necessary to use a package management system, such as conda, packrat or renv. For further guidance see: [package management](tools.html#managing-your-analytical-tools).
```

Comment:
```suggestion
In your project repository you will need a file that lists your dependencies. We use 'package management' to manage these lists. The packages change through time and may not always be backwards compatible. To avoid compatibility issues and ensure reproducible outputs, it is necessary to use a package management system, such as conda, packrat or renv. For further guidance see the [package management](tools.html#managing-your-analytical-tools) section.
```